src/mscorlib/src/System/Resources/ResourceSet.cs

```diff
@@ -182,15 +182,17 @@ public virtual Type GetDefaultReader()
             return typeof(ResourceReader);
         }
     
-#if !FEATURE_CORECLR
         // Returns the preferred IResourceWriter class for this kind of ResourceSet.
         // Subclasses of ResourceSet using their own Readers &; should override
         // GetDefaultReader and GetDefaultWriter.
         public virtual Type GetDefaultWriter()
         {
+#if FEATURE_CORECLR
+            return Type.GetType("System.Resources.ResourceWriter, System.Resources.Writer, Version=4.0.1.0, Culture=neutral, PublicKeyToken=" + AssemblyRef.MicrosoftPublicKeyToken, throwOnError: true);
```

Comment:
The ResourceWriter.cs file isn't even in the repository, IResourceWriter is though. The interface and type are both in corefx. If other public things in corelib expose IResourceWriter then we need to expose it as well.